autoapi/mappers/python/objects.py

```diff
@@ -1,6 +1,8 @@
 import collections
+import logging
 
 import sphinx
+from lazy_object_proxy.utils import cached_property
```

Comment:
Please can we remove this dependency. `display` shouldn't be getting accessed very often anyway, but even if it is we need to investigate it separately.

Reply:
As i saw in my tests, for some members `display` method called two or more times. Not very much, to be honest. I did it to prevent confusing end-user when his callback called many times for same members. As examples, user may want to count something on per-member basis and such behaviour will be surprising. As option i may cache `display` property result manually.